xapian-core/tests/api_queryparser.cc

```diff
@@ -1725,6 +1725,86 @@ DEFINE_TESTCASE(qp_range4, !backend) {
     return true;
 }
 
+static const test test_value_filesizerange1_queries[] = {
+    { "10K..20K", "0 * VALUE_RANGE 1 10240 20480" },
+    { "10..50M", "0 * VALUE_RANGE 1 10485760 52428800" },
+    { "10K..20", "Unknown range operation" },
+    { "10..20", "Unknown range operation" },
+    { "10A..20A", "Unknown range operation" },
+    { "10..20H", "Unknown range operation" },
+    { NULL, NULL }
+};
+
+// Test FileSizeRangeProcessor
+DEFINE_TESTCASE(qp_filesizerange1, !backend) {
+    Xapian::QueryParser qp;
+    Xapian::FileSizeRangeProcessor rp_filesize(1);
+    qp.add_rangeprocessor(&rp_date);
+    for (const test *p = test_value_filesizerange1_queries; p->query; ++p) {
+    string expect, parsed;
```

Comment:
These lines within the `for` loop should be indented further (and again in the next test).